### PR TITLE
MqsqlDB RDS Instance with Secrets Manager

### PR DESCRIPTION
**Pull request by koraakdas**

### CODE REVIEW

File: modules/ec2/instprep.sh [hunk 1]
```diff
@@ -6,4 +6,6 @@ systemctl enable httpd.service
 echo "The page was created by the user data" | tee /var/www/html/index.html
 yum install -y https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm
 systemctl enable amazon-ssm-agent
-systemctl start amazon-ssm-agent
\ No newline at end of file
+systemctl start amazon-ssm-agent
+yum install -y mariadb
+yum install -y unzip
```
Comment (vladimirmukhin) [inline modules/ec2/instprep.sh:11]: Not clear for me why do you want to install mariadb and unzip. Please let me know

File: modules/mysqldatabase/data.tf [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+data "aws_secretsmanager_secret" "datasource_msqldb" {
+  arn = "arn:aws:secretsmanager:us-east-1:441841492090:secret:MysqldbCreds-q7bifb"
+}
+
+data "aws_secretsmanager_secret_version" "creds" {
+  secret_id = data.aws_secretsmanager_secret.datasource_msqldb.id
+}
```
Comment (vladimirmukhin) [inline modules/mysqldatabase/data.tf:8]: Please reference secret by name instead of id. It will be more portable and easy to read

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]
Formatting is not right everywhere. Please pay closer attention to newlines and alignment of `=`

File: modules/mysqldatabase/data.tf (hunk 2 above)
Comment (koraakdas) [reply modules/mysqldatabase/data.tf:8]: done

**Review by vladimirmukhin — APPROVED** [review]

**Pull request merged**